projects/angular-test-app/tests/e2e/DigV2/ComplexFields/DataReference.spec.js

```diff
@@ -242,6 +242,22 @@ test.describe('E2E test', () => {
     deleteProduct = await page.locator('mat-chip-row:has-text("Telivision")');
     await deleteProduct.locator('button:has-text("cancel")').click();
 
+    /** Checkbox group mode type test */
+    displayAs = page.locator('mat-select[data-test-id="4aa668349e0970901aa6b11528f95223"]');
+    await displayAs.click();
+    await page.getByRole('option', { name: 'Checkbox group' }).click();
+
+    const checkbox = page.locator('app-check-box');
+    await checkbox.getByRole('option', { name: 'Washin Machine' }).click();
+    await checkbox.getByRole('option', { name: 'Mobile' }).click();
+
+    await page.locator('button:has-text("Next")').click();
+
+    await expect(assignment.locator('td >> text="Washin Machine"')).toBeVisible();
```

Comment:
Is `Washin` a typo here?

Reply:
that's the value coming from Data Page.

Reply:
We should update the Data Page in that case.